Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+click = "*"
+sentry-sdk = "*"
+
+[dev-packages]
+black = "*"
+flake8 = "*"
+bandit = "*"
+mypy = "*"
+isort = "*"
+pytest = "*"
+pytest-cov = "*"
+coveralls = "*"
+
+[requires]
+python_version = "3.9"
+
+[scripts]
+transmogrifier = "python -c \"from transmogrifier.cli import main; main()\""
```

Comment:
Do we want to consider a different, possibly shorter name for the script entry command? What about `transform`?